src/app/clusters/scenes-server/SceneTableImpl.h

```diff
@@ -31,6 +31,7 @@ static constexpr uint16_t kMaxScenesPerFabric   = CHIP_CONFIG_SCENES_MAX_PER_FAB
 static constexpr uint16_t kMaxScenesPerEndpoint = CHIP_CONFIG_MAX_SCENES_PER_ENDPOINT;
 
 using clusterId = chip::ClusterId;
+using namespace app::Clusters::Scenes::Structs;
```

Comment:
we do not mean to use a namespace here, see https://github.com/project-chip/connectedhomeip/issues/27111#issuecomment-1579571878 in the issue.

please use:
using AttributeValuePair = app::Clusters::Scenes::Structs::AttributeValuePair::Type;
using List = app::DataModel::List;
etc,
where you see fit.

Reply:
Got it thanks for the clarification.